runtime/bcverify/staticverify.c

```diff
@@ -1802,10 +1816,12 @@ j9bcv_verifyClassStructure (J9PortLibrary * portLib, J9CfrClassFile * classfile,
 							invalidRetType = TRUE;
 						}
 					}
+#if !defined(J9VM_OPT_VALHALLA_VALUE_TYPES) /* This #if !defined check should be removed (along with all other code that references J9VM_OPT_VALHALLA_NEW_FACTORY_METHOD) once the <vnew> method is introduced */
```

Comment:
is there an issue for this?

Also, can you please put the comment above the macro